packages/std-client/src/mud-definitions/world/contractComponents.ts

```diff
@@ -113,5 +113,37 @@ export function defineContractComponents(world: World) {
         }
       );
     })(),
+    KeysInTable: (() => {
+      const tableId = new TableId("", "KeysInTable");
+      return defineComponent(
+        world,
+        {
+          length: RecsType.Number,
+          keys: RecsType.BigIntArray,
+        },
+        {
+          metadata: {
+            contractId: tableId.toHexString(),
+            tableId: tableId.toString(),
+          },
+        }
+      );
+    })(),
+    UsedKeysIndex: (() => {
+      const tableId = new TableId("", "UsedKeysIndex");
+      return defineComponent(
+        world,
+        {
+          has: RecsType.Boolean,
+          index: RecsType.Number,
+        },
+        {
+          metadata: {
+            contractId: tableId.toHexString(),
+            tableId: tableId.toString(),
+          },
+        }
+      );
+    })(),
```

Comment:
As discussed in Discord, these are produced because codegen doesn't differentiate between MUD internals and modules. The other modules are ignored because codegen skips tables with `tableIdArgument`.

For now, I think we should leave these and (possibly) raise an issue for codegen to ignore them.